### PR TITLE
German Appendix, some spelling and umlaute correction

### PR DESCRIPTION
**Pull request by laeberkaes**

### CODE REVIEW

File: latex/appendix-de.tex [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\appendix
+
+\chapter{Appendix}
+
+\section{FAQ's}
+
+\section{AUTOREN}
```
Comment (ikstream) [inline latex/appendix-de.tex]: Could you please check that the people listed on the blackarch site are included?

File: latex/blackarch-guide-de.tex [hunk 2]
```diff
@@ -214,11 +223,12 @@ \subsection{Paketinstallation auf Quellcodebasis}
 \end{lstlisting}
 
 \item Blackman kann von Quellcode gebaut und installiert werden: 
+%Hier funktioniert wohl UTF8 nicht
 \begin{lstlisting}
   mkdir blackman
   cd blackman
   wget https://raw2.github.com/BlackArch/blackarch/master/packages/blackman/PKGBUILD
-  # Sicherstellen dass die PKGBUILD nicht bösartig verändert worden sind.
+ # Sicherstellen, dass die PKGBUILD nicht boesartig veraendert worden sind.
```
Comment (ikstream) [inline latex/blackarch-guide-de.tex]: It looks like you removed the Umlaute

File: latex/blackarch-guide-de.tex [hunk 3]
```diff
@@ -16,14 +16,22 @@
 \usepackage{fancyvrb}
 \usepackage{geometry}
 \usepackage{graphicx}
-\usepackage{html}
+%\usepackage{html}
```
Comment (ikstream) [inline latex/blackarch-guide-de.tex]: Please don't use comments here. If it is not needed, remove it.

File: latex/blackarch-guide-de.tex [hunk 4]
```diff
@@ -214,11 +223,12 @@ \subsection{Paketinstallation auf Quellcodebasis}
 \end{lstlisting}
 
 \item Blackman kann von Quellcode gebaut und installiert werden: 
+%Hier funktioniert wohl UTF8 nicht
```
Comment (ikstream) [inline latex/blackarch-guide-de.tex]: Please remove unneeded comments. This does not add any value

**Comment by laeberkaes** [comment]
I made the canges you requested. I also translated the FAQs from the Website. Can you have a look?

**Comment by ikstream** [comment]
There is just the one thing with the Autoren section left. Could you please add the authors from the blackarch site?

**Comment by ikstream** [comment]
I am sorry, i haven't seen your update. I will try to build and merge afterwards

**Comment by ikstream** [comment]
Thank you for your contribution

**Pull request merged**